src/components/stage-header/stage-header.jsx

```diff
@@ -78,51 +80,54 @@ const StageHeaderComponent = function (props) {
             </Box>
         );
     } else {
+        const stageControls =
+            mode === 'editor' ? (
```

Comment:
Here we should compare against one of the `MODES` in case this string `'editor'` changes for some reason. But I think better would be if we didn't import that enum at all (which I would consider internal to the reducer) and instead imported a helper function like `isPlayerMode(state.guiMode.mode)`.  And even better would be if we used that in `mapStateToProps` and just had `isPlayerMode` as a boolean prop wherever we needed to know about the mode.

Reply:
+1 on the isPlayerMode boolean. I just realized that mode was when I thought there would be three modes (editor, player, fullscreen) and just kept it due to inertia when I pulled out fullscreen.